sdk/dotnet/Pulumi/Core/InputList.cs

```diff
@@ -55,19 +55,29 @@ private InputList(Output<ImmutableArray<T>> values)
         {
         }
 
-        public void Add(params Input<T>[] inputs)
```

Comment:
Changing this is a binary breaking change. Let's keep the existing params overload.